catroid/src/org/catrobat/catroid/ui/SettingsActivity.java

```diff
@@ -94,6 +95,18 @@ protected void onCreate(Bundle savedInstanceState) {
 		setNXTSensors();
 		setDronePreferences();
 
+		CheckBoxPreference pref = (CheckBoxPreference) findPreference(SETTINGS_SHOW_PARROT_AR_DRONE_BRICKS);
```

Comment:
Please clarify. The name of the quadcopter is [Parrot AR.Drone](https://en.wikipedia.org/wiki/Parrot_AR.Drone)